src/snowflake/snowpark/modin/plugin/__init__.py

```diff
@@ -97,6 +97,22 @@
 for (doc_module, target_object) in inherit_modules:
     _inherit_docstrings(doc_module, overwrite_existing=True)(target_object)
 
+# _inherit_docstrings needs a function or class as argument, so we must explicitly iterate over
+# all members of io and general
+for name in dir(modin.pandas.io):
```

Comment:
Should we rewrite these two loops as another loop over `(modin.pandas.io, modin.pandas.general)`?

Reply:
Sure, done.